community/kernel/src/main/java/org/neo4j/kernel/impl/store/StoreType.java

```diff
@@ -0,0 +1,234 @@
+/*
+ * Copyright (c) 2002-2015 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.kernel.impl.store;
+
+import java.io.IOException;
+
+import org.neo4j.graphdb.factory.GraphDatabaseSettings;
+import org.neo4j.kernel.IdType;
+import org.neo4j.kernel.impl.store.counts.CountsTracker;
+
+public enum StoreType
+{
+    NODE_LABEL( StoreFactory.NODE_LABELS_STORE_NAME )
+            {
```

Comment:
I've tried fixing that for years, I even reported an issue about it to the Intellij guys, to no use. With the opening curly on a new line Intellij (incorrectly) treats this as a "continuation line" and indents as such.